README.md

```diff
@@ -56,6 +60,8 @@ $ bundle exec npmdc [options]
 _Options:_
 
 ```bash
+      [--package_manager]      # Possible values: npm, yarn; npm by default
+      [--path_to_yarn]         # Path to your yarn installation; nil by default
```

Comment:
Let's use kebab-case for CLI options, i.e. `--path-to-yarn`, `--package-manager`.